cabra/src/pages/components/TaskListItem.tsx

```diff
@@ -44,8 +46,15 @@ export default function TaskListItem({ task }: Props) {
     }
   };
 
+  const isSubtask = task.relationTypes.includes(DirectedRelationType.Subtask);
+
   return (
-    <div tw="rounded-xl w-full bg-primary text-stone-50 shadow-2xl p-4 grid grid-cols-[1fr 30%] gap-x-4">
+    <div
+      css={[
+        tw`rounded-xl w-full text-stone-50 shadow-2xl p-4 grid grid-cols-[1fr 30%] gap-x-4`,
+        isSubtask ? tw`bg-primary bg-opacity-60` : tw`bg-primary`,
+      ]}
+    >
```

Comment:
I think coloring the subtask is not necessary. We have an icon, it's enough information.